packages/modules/http/device.py

```diff
@@ -117,11 +115,18 @@ def read_legacy(argv: List[str]) -> None:
     component_config["id"] = num
     dev.add_component(component_config)
 
-    log.MainLogger().debug('Http Konfiguration: ' + str(component_config["configuration"]))
+    log.MainLogger().debug(
+        'Http Konfiguration: ' + str(device_config["configuration"]) + str(component_config["configuration"]))
 
     dev.update()
 
 
+def __parse_arg(arg):
+    if arg != "none":
+        arg = arg.replace(re.search("http[s]?://[0-9.]+", arg).group(), "")
```

Comment:
Funktioniert nur mit IPv4-Adressen. Also kein IPv6 und keine Hostnames.

Gegenvorschlag:
```python
def _parse_arg(arg: str):
    return re.search("^(?:https?://[^/]+)?(.*)", arg).group(1)
```
Demo:
```
>>> _parse_arg("none")
'none'
>>> _parse_arg("/relative")
'/relative'
>>> _parse_arg("http://domain.tld/path")
'/path'
>>> _parse_arg("https://domain.tld/path")
'/path'
>>> _parse_arg("https://123.456.789.012/path")
'/path'
```

Reply:
Besser als `_parse_arg` wäre als Name der Funktion noch `extract_url_path` oder sowas in der Art. Das gibt eher eine Idee was es tut.